cms/static/js/views/import.js

```diff
@@ -2,241 +2,306 @@
  * Course import-related js.
  */
 define(
-    ["domReady", "jquery", "underscore", "gettext"],
-    function(domReady, $, _, gettext) {
+    ["jquery", "underscore", "gettext", "moment", "jquery.cookie"],
+    function($, _, gettext, moment) {
 
         "use strict";
 
-        /********** Private functions ************************************************/
+        /********** Private properties ****************************************/
+
+        var STATE = {
+            'READY'      : 1,
+            'IN_PROGRESS': 2,
+            'SUCCESS'    : 3,
+            'ERROR'      : 4
+        }
```

Comment:
@AkA84 nit: missing semicolon here, on line 37 (`destroyEventListeners`), 57 (`initEventListeners`) and 174 (`storeImport`)